### PR TITLE
Issue 11 v2

### PR DESCRIPTION
**Pull request by kevinsrose**
screen shot 2017-09-17 at 5 58 17 pm
Sidebar should now be resizable with click & drag
Also refactored app.js slightly to improve clarity of module, config, etc.

### CODE REVIEW

File: src/cacheDisplay.html [hunk 1]
```diff
@@ -94,114 +92,110 @@
...
-                    <md-button class="md-icon-button"><i class="material-icons">replay</i></md-button>
-                </md-content>
-                <md-content layout="row" layout-align="center" style="padding-top: 25px; padding-right: 10px;">
-                    <div layout layout-align="center center">
-                        <span class="md-body-1">Speed</span>
-                    </div>
-                    <md-slider flex class="md-primary" md-discrete ng-model="$ctrl.speedRating" step="1" min="1" max="5" aria-label="rating">
-                    </md-slider>
-                </md-content>
+            <!-- Control Flow Buttons -->
```
Comment (jbonzo) [inline src/cacheDisplay.html:180]: what were all these changes? was it just white spacing?

Comment (kevinsrose) [reply src/cacheDisplay.html:180]: Yeah everything is mostly the same except for the section tag at the top, this was all previously embedded in a body tag that I removed so it changed the indentation for everything

**Comment by jbonzo** [comment]
Just two UI things:
* I think a thin line would be beneficial as a divider between the cache displays (left) and the other side (right). took me a second to find the resize button and it would be a nice visual separator between these two functional panes
* its a little hard to drag it. If I click and drag to move it left, then click again it'll reset back to position zero. Also it lags behind my mouse

Let me know if this makes sense

**Comment by kevinsrose** [comment]
Yeah I was thinking we should add some sort of border between the two for separation but I think that will require some refactoring of the html and should warrant a separate issue for it. I noticed the click and drag is a little off I think there is a way to change the "resize speed" that I'll look into, for now I was mainly concerned with the functionality

**Comment by jbonzo** [comment]
do you see what I mean about the reset though?

**Comment by jbonzo** [comment]
But yeah as far as functionality goes the reset thing is the only thing holding it back.

**Comment by kevinsrose** [comment]
I didn't notice the reset thing, I'll look into it tomorrow morning

**Review by jbonzo — APPROVED** [review]
looks sick

**Comment by kevinsrose** [comment]
Issue should be fixed now, the click & drag lagging behind the mouse and the 'reset' were being caused by the same thing

**Pull request merged**